soc/silabs/silabs_siwx91x/siwg917/siwx91x_nwp.c

```diff
@@ -467,7 +507,13 @@ BUILD_ASSERT(CONFIG_SIWX91X_NWP_INIT_PRIORITY < CONFIG_KERNEL_INIT_PRIORITY_DEFA
 	static const struct siwx91x_nwp_config siwx91x_nwp_config_##inst = {                       \
 		.config_irq = silabs_siwx91x_nwp_irq_configure_##inst,                             \
 		.power_profile = DT_ENUM_IDX(DT_DRV_INST(inst), power_profile),                    \
-		.stack_size = DT_INST_PROP(inst, stack_size)                                       \
+		.stack_size = DT_INST_PROP(inst, stack_size),                                      \
+		.rtc_from_host = DT_PROP_OR(DT_DRV_INST(inst), rtc_from_host, false),              \
+		.support_1p8v = DT_PROP_OR(DT_DRV_INST(inst), support_1p8v, false),                \
+		.disable_xtal_correction = DT_PROP_OR(DT_DRV_INST(inst),                           \
+						      disable_xtal_correction, false),             \
+		.qspi_80mhz_clk = DT_PROP_OR(DT_DRV_INST(inst), qspi_80mhz_clk, false),            \
+		.antenna_selection = DT_PROP_OR(DT_DRV_INST(inst), antenna_selection, "none"),     \
```

Comment:
You don't need `DT_PROP_OR()`. From `api-usage.rst`:

>   Use DT_PROP() [...] will expand to either 0 or 1 depending on if the property
>   is present or absent.

You can even use `DT_INST_PROP()` so you won't need `DT_DRV_INST()`.